### DISCUSSION TITLE
E355: Unknown option: winbar

### DISCUSSION THREAD

**Discussion by greensourcecode** [crivotz/nv-ide#16] [Q&A] [answered]
I have ran into this issue on multiple different systems. Does anyone know what this is / if it matters / how to fix it?

![error](https://user-images.githubusercontent.com/97241812/174410028-1f8f1a5f-4600-4322-a156-3048d2e473ef.png)

**Comment by crivotz** [comment, marked as answer]
winbar is a new feature of neovim present in the nightly version. I see to make a branch with the version without winbar ;)

**Reply by crivotz** [reply]
Ok, you can use branch neovim_stable ;)
Sorry for the inconvenience
